google/ggrc-core
# GGRC-7295 Update Default Assignees and Default Verifiers lists for Assessment template

# Issue description
Default Assignees and Default Verifiers fields are empty after create Assessment template with "Risk Owners" roles

# Steps to test the changes

Create Assessment Template for Risk and select Default Assignee - Default Verifiers fields with "Risk owner" roles
Check Assessment Template after creation
"Risk owner" should be showed for Default Assignee and Default Verifier 

# Solution description
Update constant with "Risk Owner" role for Assessment Template

# Sanity checklist


# PR Review checklist
